src/helperFunctions.js

```diff
@@ -163,11 +165,13 @@ helpers.prototype._makeOCSrequest = function (method, service, action, data) {
   if (service) {
     slash = '/'
   }
-
   var path = this.OCS_BASEPATH + service + slash + action
+  if (options.useV2) {
```

Comment:
should we use V2 by default ?

Reply:
I will step by step migrate this. but the current implementation of makeOCSRequest is relying on v1 to be used. I'll drop the v2 part ... it did not work.

THX for a second pair of eyes - I missed that